tasks/main.yml

```diff
@@ -33,8 +33,13 @@
   notify:
     - Restart Rsyslog
 
+- debug: msg="Adding papertrail config at {{ papertrail_remote_syslog_config_paths[ansible_service_mgr].path }}"
+
 - name: Configure Papertrail service definition
-  template: src="remote_syslog.conf.j2"
-            dest="/etc/init/remote_syslog.conf"
+  template: src="remote_syslog.{{ ansible_service_mgr }}.conf.j2"
+            dest="{{ papertrail_remote_syslog_config_paths[ansible_service_mgr].path }}"
+
+- name: Enable Papertrail service
+  service: name=remote_syslog enabled=true
   notify:
```

Comment:
I added it here because even though the service is configured in the step before, systemd doesn't recognize it until it's explicitly enabled. This seemed like a good compromise to keep the playbook service-manager agnostic without adding more conditional tasks.